R/resource.R

```diff
@@ -265,8 +265,21 @@ hipercow_resources_validate <- function(resources,
                                         driver = NULL,
                                         root = NULL) {
 
+  # A bit of a hack for now - handle NULL driver (which often happens
```

Comment:
can you make a ticket with in the "debt and maintenance" epic?

Reply:
https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-4942